gef.py

```diff
@@ -9146,6 +9146,14 @@ def __gef_prompt__(current_prompt):
             "Consider updating to GDB {} or higher.".format(".".join(GDB_MIN_VERSION)))
 
     else:
+        import commands
+        import site
+        if commands.getstatusoutput('pyenv root'):
```

Comment:
Why not use subrocess?

Reply:
<img width="1159" alt="screen shot 2018-10-13 at 3 03 02 am" src="https://user-images.githubusercontent.com/13882826/46886020-9419ca80-ce94-11e8-9f5a-d69d0f4c351a.png">

I changed to use subprocess.